src/llmcompressor/modeling/deepseek_v3.py

```diff
@@ -0,0 +1,48 @@
+import torch
+from transformers.models.deepseek_v3.modeling_deepseek_v3 import DeepseekV3MoE
+
+
+class DeepseekV3MoECalibrate(torch.nn.Module):
```

Comment:
a docstring/comment indicating what's different between this and original in `transformers.models.deepseek_v3.modeling_deepseek_v3` would be helpful